src/js/helpers/Util.js

```diff
@@ -0,0 +1,15 @@
+/*eslint-disable no-alert */
```

Comment:
Yeah this looks pretty good! Could we replace `window` with `global` and because I can see this `Util` growing as time goes, could we do something like this so that we are a little more explicit on what we are linting:
```
    alert: function (msg) {
      /*eslint-disable no-alert */
      return window.alert(msg);
      /*eslint-enable no-alert */
    },
    confirm: function (msg) {
      /*eslint-disable no-alert */
      return window.confirm(msg);
      /*eslint-enable no-alert */
    },
    ...
```